components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/DestinationEntity.java

```diff
@@ -0,0 +1,52 @@
+package com.sap.cloud.cmp.ord.service.storage.model;
+
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmIgnore;
+import com.sap.olingo.jpa.metadata.core.edm.annotation.EdmProtectedBy;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import org.eclipse.persistence.annotations.Convert;
+import org.eclipse.persistence.annotations.TypeConverter;
+
+import java.util.Set;
+import java.util.UUID;
+
+@Entity(name = "destination")
+@Table(name = "tenants_destinations")
```

Comment:
Make sure to adopt the optimizations in the view after merged:

compass: https://github.com/kyma-incubator/compass/pull/2582
ord: https://github.com/kyma-incubator/ord-service/pull/78